autometa/common/external/work_queue.py

```diff
@@ -144,7 +144,7 @@ def main(args):
     q.activate_fast_abort(2)
 
     # create task as usual and tag it with an arbitrary string.
-    t = wq.Task(...)
+    # t = wq.Task(...)
```

Comment:
So I haven't gone through this script yet. Could you perhaps comment on what is going on here, and the reason for this change?

Reply:
This script is not functional right now. It will start up a workqueue master then raise a NotImplementedError